libctru/include/3ds/thread.h

```diff
@@ -23,7 +23,7 @@ typedef struct Thread_tag* Thread;
  * @param affinity The ID of the processor the thread should be ran on. Processor IDs are labeled starting from 0.
  *                 On Old3DS it must be <2, and on New3DS it must be <4.
  *                 Pass -1 to execute the thread on all CPUs and -2 to execute the thread on the default CPU (read from the Exheader).
- * @param detached When set to true, the thread is automatically freed when it finishes.
+ * @param detached When set to true, the thread is automatically freed when it finishes. Exit code will be 0.
```

Comment:
This comment should actually be another note, since it is always true. I'd like to change it to `@note Exit code will be 0 if threadExit isn't called.`, what do you think ?